corehq/apps/reports/tests/test_application_status_report.py

```diff
@@ -0,0 +1,465 @@
+                        'commcare_version': '2.48.0'
+                    },
+                    'last_submissions': [
+                        {
+                            'app_id': 'app1',
+                            'submission_date': format_datetime(now - timedelta(days=2)),
+                            'commcare_version': '2.53.0'
+                        }
+                    ],
+                    'last_syncs': [
+                        {
+                            'app_id': 'app1',
+                            'sync_date': format_datetime(now - timedelta(days=2))
+                        }
+                    ],
+                    'last_sync_for_user': {
+                        'sync_date': format_datetime(now - timedelta(days=12))
+                    },
+                    'last_build_for_user': {
+                        'app_id': 'app1',
+                        'build_version': 7
+                    }
+                },
+                'devices': [
+                    {
+                        'device_id': 'device3',
+                        'last_used': format_datetime(now),
+                        'app_meta': [
+                            {
+                                'app_id': 'app1',
+                                'num_unsent_forms': 2
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+
+
+@es_test(requires=[user_adapter], setup_class=True)
+@disable_quickcache
+class TestApplicationStatusReport(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Clean up any existing domains and users
+        delete_all_users()
+        for domain in Domain.get_all():
+            domain.delete()
+
+        cls.domain_name = 'app-status-test'
+        cls.domain = create_domain(cls.domain_name)
+        cls.addClassCleanup(cls.domain.delete)
+        cls.request_factory = RequestFactory()
+
+        # Create test users
+        cls.admin_user = WebUser.create(cls.domain_name, 'admin@example.com', 'password', None, None)
+        cls.admin_user.is_superuser = True
+        cls.admin_user.save()
+        cls.addClassCleanup(cls.admin_user.delete, cls.domain_name, deleted_by=None)
+        # Create location data first
+        cls.location_type = LocationType.objects.create(
+            domain=cls.domain_name,
+            name='facility',
+        )
+        cls.location = SQLLocation.objects.create(
+            domain=cls.domain_name,
+            name='Test Facility',
+            location_id='location1',
+            location_type=cls.location_type
+        )
+        # Create test data
+        cls.user_data = TestApplicationStatusReportData.get_user_data(cls.domain_name)
+        cls.users = []
+
+        # Now create users with proper metadata
+        for user_data in cls.user_data:
+            # Create the user properly
+            user = CommCareUser(user_data)
```

Comment:
Since one user have access to apps in different domain can only happen for WebUser, does it make more sense to make it WebUser in the test?